apps/src/constants.js

```diff
@@ -26,6 +26,7 @@ exports.TestResults = {
   NO_TESTS_RUN: -1,
 
   // The level was not solved.
+  GENERIC_FAIL: 0,                    // Used by DSL defined levels.
```

Comment:
This is the value that `multi` was posting when the answer result is `true`. I'm trying to match the testResult value but using a constant instead of doing `result ? 100: 0` like `multi` did.